packages/react-app/src/components/chart.js

```diff
@@ -23,60 +22,17 @@ function dateToUTCEpoch(d) {
     return Number(Math.round(utcMilllisecondsSinceEpoch / 1000))
 }
 
-function oneYearAgo() {
+function calculatePastEpoch(months_ago) {
```

Comment:
maybe functions like this should go in here?
https://github.com/nucypher/nuit/blob/main/packages/react-app/src/services/nucypher.js